src/x25519.rs

```diff
@@ -121,13 +121,45 @@ fn clamp_scalar(scalar: [u8; 32]) -> Scalar {
 ///
 /// This can be used with [`X25519_BASEPOINT_BYTES`] for people who
 /// cannot use the better, safer, and faster ephemeral DH API.
+/// # Example
+/// ```
+/// extern crate rand_os;
+///
+/// use x25519_dalek::{ x25519, X25519_BASEPOINT_BYTES };
+/// use rand_os::OsRng;
+/// use rand_os::rand_core::RngCore;
+///
+/// let mut rng = OsRng::new().unwrap();
+///
+/// // Generate Alice key pair
+/// let mut alice_private = [0u8; 32];
```

Comment:
The private keys aren't exactly 32 random bytes… why not use the `EphemeralSecret` type?

Reply:
Hi!
It has been requested in some issues to have a way to get the underlying bytes of the secret key, as some cases cannot be dealt with with an opaque structure(my specific case if for cross-language support using FFI).
The function x25519() is exposed for people who wants the RFC-defined implementation on bare bytes, but there isn't any example usage for this. 
So, for this PR, I made and example using this that works from a user perspective, but I might have missed something on how the private key is generated? I though that it was simply a random 32 bytes scalar value.

Reply:
I just checked closed issues and it seems like the new StaticSecret structs implemented in 0.5.0 now works with my specific use case. However, I believe it would still be worthwhile to document a good use of the x25519() method if someone else ends up needing it. What do you think?

Reply:
I feel like it might be better to only facilitate usage of the StaticSecret type, since in Rust that will at least guarantee that the secrets are zeroed out from memory when they fall out of scope (so in many cases, when an FFI boundary is crossed).